src/adapter/src/repositories/grpc/models/gpt_answer.rs

```diff
@@ -0,0 +1,40 @@
+use rust_core::common::errors::CoreError;
+use tonic::transport::Channel;
+
+use grpc_interface::interfaces::gpt_answer::gpt_answer::{
+    gpt_answer_service_client::GptAnswerServiceClient, GetAnswerPayload,
+};
+
+pub struct GptAnswerGrpcClient {
+    client: GptAnswerServiceClient<Channel>,
+}
+
+impl GptAnswerGrpcClient {
+    fn new(channel: Channel) -> Self {
+        let client = GptAnswerServiceClient::new(channel);
+        Self { client }
+    }
+
+    pub async fn get_instance(uri: &'static str) -> Result<Self, CoreError> {
+        let channel = Channel::from_static(uri).connect().await.map_err(|err| {
+            eprintln!("Error connecting to GPT: {:?}", err);
```

Comment:
Replace this and implement something like this:
```rust
CoreError::InternalError(err)
```
And then in the error handle function:
Print the error to the logger;
Print the error inside the response if server is running in debug mode